common/goal_distance_calculator/src/goal_distance_calculator_node.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Tier IV, Inc. All rights reserved.
+// Copyright 2020-2023 Foxconn, Tier IV, Inc. All rights reserved.
```

Comment:
@PhoebeWu21 Copy right should not be changed at a small change PR and this PR will not change behavior.

https://github.com/autowarefoundation/autoware.universe/pull/4730#discussion_r1304345840

Reply:
ok!